tests/test_parse_doc.m

```diff
@@ -0,0 +1,19 @@
+function test_parse_doc
+%TEST_PARSE_DOC Is the unit test corresponding to parse_doc.m
+%
+% DESCRIPTION
+%
+% That function makes sure that parse_doc is working properly and returns
+% the expected structure for the documentation.
+cd('../parser')
```

Comment:
There will be an error if the function is run from another directory than the one tested (and in Matlab, it is possible to run a script or function from a different directory). Maube you should check with a pwd funciton end extrapolate the correct way to get to the place you want. You could also check this link to know more about finding an item location in Matlab:
[https://www.mathworks.com/help/matlab/ref/which.html](url)
This way your code could be more robust.

Reply:
I think you should consider removing this line entirely and make the code work whether this is called from anywhere.

Reply:
Yes that's something I why trying to do but I could only find the path to the function using which if I was already in the right folder (not really useful in our case...). One solution would be to add `helpDocMd/parser/` to the Matlab path but It would also require to know the full path toward it. @po09i do you know how this is usually done with unit testing ?

Reply:
The which command in Matlab does this. It returns the absolute path of the file you asked. See the link I added in my previous comment.

Reply:
There should be a startup.m script like the [one](https://github.com/shimming-toolbox/shimming-toolbox/blob/master/startup.m) in shimming-toolbox which adds all the source files to the Matlab path. This script would be launched when running the tests. So my point of view is you should assume all of helpDocMd is on the Matlab path. If it is not yet created in this repo, I think you should create it.